selfdrive/ui/qt/offroad/onboarding.cc

```diff
@@ -121,7 +121,16 @@ void OnboardingWindow::updateActiveScreen() {
   }
 }
 
+void enableBacklight(){
```

Comment:
this shouldn't go in onboarding. ideally we want a nice HW abstraction in cpp like we have in python

Reply:
it should also be gated on QCOM2

Reply:
It's also only a problem for testing when you keep restarting the ui. On boot the backlight is already set to max.